### PR TITLE
Add code formatting rules and format the existing code [WIP]

### PR DESCRIPTION
**Pull request by nemanjaglumac**
#### Changes
This PR adds some sane defaults re: code formatting. It attempts to explicitly communicate and set the code style rules for the whole project.

I've envisioned three stages for this PR:
1. Add `.prettierc` and **define** all the rules (code review process should be used for discussion)
2. Install Prettier as a dev-dependency and create a script that would format the code
3. Finally, commit the formatted code to the code base as the new baseline

> **Warning**
> Formatting the code with these new rules will create a huge diff. It is somewhat painful step and it is crucial to get it done as early as possible for that very reason.

#### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update (quite possibly, not sure yet)

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"trailingComma": "es5",
+	"semi": true,
+	"singleQuote": false,
+	"no-bracket-spacing": false,
```
Comment (nemanjaglumac) [inline .prettierrc:5]: Also inferred from the existing code.

**Comment by detronetdip** [comment]
@nemanjaglumac for lint checking I think super-linter is a very good option

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+	"trailingComma": "none",
+	"semi": true,
+	"singleQuote": false,
+	"bracketSpacing": true,
```
Comment (nemanjaglumac) [inline .prettierrc:5]: Also inferred from the existing code.

File: package.json [hunk 3]
```diff
@@ -2,12 +2,16 @@
 	"name": "youperiod.app",
 	"version": "0.0.1",
 	"scripts": {
-		"start": "node server.js"
+		"start": "node server.js",
+		"prettify": "prettier --write ."
```
Comment (nemanjaglumac) [inline package.json:6]: Alternatively, explicitly pass the files to format like so:
```suggestion
		"prettify": "prettier --write server.js web"
```

**Comment by getify** [comment]
We can definitely revisit as the project starts to evolve and grow its contributor base (and code complexity). :)

**Comment by getify** [comment]
Question: is it possible to have some sort of "anti-config" that, for example, disables the built-in user-level linting and code formatting that people may have set up in their own editors? That way it would cut down on the annoyance some may find if they forget to manually disable such stuff?

**Comment by getify** [comment]
@nemanjaglumac I think that your suggestion makes sense. Please also include text asking contributors to also attempt to "fit" (not necessarily exactly match) as closely as reasonable the existing code styles and conventions, just to reduce some of the wildly diverging styles that might come about.

**Pull request closed without merging**